### PR TITLE
update default migration template

### PR DESCRIPTION
**Pull request by RandomSeeded**

### CODE REVIEW

**Comment by RandomSeeded** [comment]
DO NOT MERGE THIS WITHOUT NATE'S APPROVAL

**Review by jamesongithub — APPROVED** [review]
lgtm if tested

**Comment by RandomSeeded** [comment]
screen shot 2018-02-27 at 7 19 37 am

On my local

**Pull request merged**
